core/health/rest/src/main/java/org/opennms/core/health/rest/HealthCheckRestService.java

```diff
@@ -46,5 +46,5 @@
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    Response getHealth(@QueryParam("t") @DefaultValue("5000") int timeoutInMs);
+    Response getHealth(@QueryParam("t") @DefaultValue("5000") int timeoutInMs, @Context final UriInfo uriInfo);
```

Comment:
Can we derivce `@QueryParam("t")`  from UriInfo ?

Reply:
Technically we can. But I'd prefer not to do that. Typically, if we can use Query Param, we should avoid using the UriInfo. I think our code in other places does follow that pattern.